Doc/library/struct.rst

```diff
@@ -290,6 +290,11 @@ Notes:
    operations. See the Wikipedia page on the `half-precision floating-point
    format <half precision format_>`_ for more information.
 
+(7)
+   The "long integer" type is defined as a 4 byte type on both Windows OSs and
```

Comment:
Let's refer to the literal name of the type, ``long``.
```suggestion
   The :c:type:`long` or "long integer" type is defined as a 4 byte type on both Windows OSs and
```

Note that this change will require adjusting the wrapping of the paragraph to fit within the 80 char maximum line width for reST.

Reply:
I think this statement is too specific: it describes the *current* state on common operating systems (macOS, Windows, Linux on x86 or x64 hardware), but since it's describing something that's out of CPython's control, and that could change in the future, and there may well be existing systems for which the statement isn't true. There's no reason that a Unix system couldn't use an LLP64 data model instead of LP64.